apps/examples/log_dump/log_dump_main.c

```diff
@@ -57,6 +66,14 @@ int log_dump_main(int argc, char *argv[])
 			lldbg_noarg("%c", buf[i]);
 		}
 	}
+
+	if (write(fd, LOGDUMP_SAVE_STOP, strlen(LOGDUMP_SAVE_STOP) + 1) < 0)
```

Comment:
How about providing some wrapper APIs instead of raw usages of write function like reboot reason?

Reply:
Added API like below:
```
START_LOGDUMP_SAVE(fd);
STOP_LOGDUMP_SAVE(fd);
```